tests/bsim/bluetooth/host/l2cap/fixed_custom_chan/prj.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_BT=y
```

Comment:
Note: Building images is what takes the longest CI time.

This image seems to not need anything specially weird. Would it be possible to have this test be built together with some other test image?